Portfolio desktop version

shahira-sadat opened the pull request:
In this pull, I have worked on portfolio desktop version using media queries.
there is two files in my css folder one is style.css which is for mobile version styling and the second one is general.css. 
In general.css you can find desktop version styles.
After the first review I modified my files and made needed changes.
- used flex and grid
- fixed card positions problems

Kindly review it and share your feedback with me.
Thanks in advance

Whoistolu reviewed (changes requested):
Hi @shahira-sadat 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Kandy-Peter commented on css/general.css:
```diff
@@ -105,77 +112,192 @@
+    gap: 2em;
+  }
+
+  .card1 {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: center;
+    background: #fff;
+    border: 2px solid #a7aeff;
+    border-radius: 24px;
+    width: 75vw;
+    height: 58vh;
+    gap: -1em;
+  }
+
+  .card-image {
+    border-radius: 8px;
+    background-repeat: no-repeat;
+    background-size: 100% 78%;
+    margin-left: 10px;
+    margin-top: 20px;
+    width: 580px;
+    height: 500px;
+  }
+
+  .card-image1 {
+    background-image: url("../image/Snap3.png");
+  }
+
+  .card-image2 {
+    background-image: url("../image/Snap2.png");
+    margin-right: 30px;
```
- [ ] As suggested by the previous reviewer, You should implement your code to avoid child elements overflow. I noticed you tried to fix that issues but it broke some points and your image and text inside your cards have exceeded their parent card div. Please fix these issues to make your design match the Figma template.
### Hint:

- I saw that you used all card images as `background-image`. However, to avoid these issues, It's good to add them as HTML elements using the `<img>` tag and use either `flexbox` or `grid` to align them with their sibling's elements.
After that, you can use the CSS properties `object-fit: cover` and `overflow: hidden` to ensure that your image is well fixed in its container.

**1. Your cards**
![image](https://user-images.githubusercontent.com/80612925/171937715-221b4efa-2b5d-4120-bd29-2c3ec03f0ee7.png)

**2. The Figma desing**
![image](https://user-images.githubusercontent.com/80612925/171937573-8d81ad2c-c807-4399-9326-639ef96a7a4c.png)

Kandy-Peter commented on index.html:
```diff
@@ -253,41 +282,41 @@ <h3>Languages</h3>
+            that you need built or a project that needs coding. I’d love to help
+            with it
+          </p>
+          <div class="footer-items">
+            <form action="https://formspree.io/f/xjvlllyy" method="POST">
+              <input
+                name="name"
+                id="user-name"
+                type="text"
+                placeholder="Name"
+                maxlength="30"
+                required
+              />
+  
+              <input
+                name="email"
+                id="user-email"
+                type="email"
+                placeholder="name@gmail.com"
+                required
+              /><textarea
+                name="message"
+                maxlength="500"
+                required
+                placeholder="Write your message here"
+              ></textarea>
+              <button type="submit">Get in Touch</button>
+            </form>
+            <img class="footer-img" src="image/Shape.png" alt="footer-image" />
+        </div>
+        </div>
+      </footer>
```
- [ ] It would be better if you enlarge your contact form to fit your [Figma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). THis will enhance the beauty of your design 😄. I know you can make it better 😊 
1. **Your design** 
![image](https://user-images.githubusercontent.com/80612925/171938875-abdfc9d3-e287-4518-98f3-85bc5de3fcf2.png)
2. **The Figma design**
![image](https://user-images.githubusercontent.com/80612925/171939199-4bb7875c-a849-46ea-83a1-3f8701e4e69e.png)

Kandy-Peter commented on index.html:
```diff
@@ -253,41 +282,41 @@ <h3>Languages</h3>
+            that you need built or a project that needs coding. I’d love to help
+            with it
+          </p>
+          <div class="footer-items">
+            <form action="https://formspree.io/f/xjvlllyy" method="POST">
+              <input
+                name="name"
+                id="user-name"
+                type="text"
+                placeholder="Name"
+                maxlength="30"
+                required
+              />
+  
+              <input
+                name="email"
+                id="user-email"
+                type="email"
+                placeholder="name@gmail.com"
+                required
+              /><textarea
+                name="message"
+                maxlength="500"
+                required
+                placeholder="Write your message here"
+              ></textarea>
+              <button type="submit">Get in Touch</button>
+            </form>
+            <img class="footer-img" src="image/Shape.png" alt="footer-image" />
+        </div>
+        </div>
+      </footer>
```
- [ ] It would be better if you enlarge your contact form to fit your [Figma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) by implementing those `sizes`, `font` etc. THis will enhance the beauty of your design 😄. I know you can make it better 😊 
1. **Your design** 
![image](https://user-images.githubusercontent.com/80612925/171938875-abdfc9d3-e287-4518-98f3-85bc5de3fcf2.png)
2. **The Figma design**
![image](https://user-images.githubusercontent.com/80612925/171939199-4bb7875c-a849-46ea-83a1-3f8701e4e69e.png)

shahira-sadat replied:
Hi, Thanks for your review @Kandy-Peter 
I worked on required changes and this is my last review chance, so it will be pleasure if you check it again.
- I modified style for contact form as required.
- I also modified some style in card
About cards, everything looks good in my side. kindly check this screenshot.

<img width="1440" alt="Screen Shot 2022-06-04 at 3 34 18 AM" src="https://user-images.githubusercontent.com/53530780/171965463-500bb095-2df4-4501-8393-f16ad67417e5.png">

waiting for your feedback and thanks in advance.

shahira-sadat replied:
Hi, Thanks for your review @Kandy-Peter 
I worked on required changes and I have just one another review chance,
soI will be very grateful if you check it again.
- I modified style for contact form as required.
- I also modified some styles in card

About cards, everything looks good in my side. 
waiting for your feedback and thanks in advance.

Kandy-Peter reviewed (changes requested):
Hi @shahira-sadat ,

Great work on making the changes requested by a previous reviewer 👏
You've done well implementing some of the requested changes, but there are still some issues which aren't been addressed yet.

![image](https://user-images.githubusercontent.com/80612925/169586832-56418ba8-59ea-4376-97ab-65fb0f51c869.png)

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required Changes ♻️
- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)..

shahira-sadat commented:
I worked on required changes and this is my last review chance.
- I modified style for contact form as required.
- I also modified some style in card

About cards, everything looks good in my side.  I hope to get approval for my last chance.
Thanks in advance for your review

Ginohmk reviewed (approved):
Hi @shahira-sadat , 

# STATUS: APPROVED 🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

![well-1](https://user-images.githubusercontent.com/64173189/128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c.gif)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
